example/tutorials/t8_step2_uniform_forest.cxx

```diff
@@ -0,0 +1,183 @@
+
+  return forest;
+}
+
+/* Write vtk (or more accurately vtu) files of the forest.
+ * \param [in] forest   A forest.
+ * \param [in] prefix   A string that is used as a prefix of the output files.
+ * 
+ * This will create the file prefix.pvtu
+ * and additionally one file prefix_MPIRANK.vtu per MPI rank.
+ */
+static void
+t8_step2_write_forest_vtk (t8_forest_t forest, const char *prefix)
+{
+  t8_forest_write_vtk (forest, prefix);
+}
+
+/* Destroy a forest. This will free all allocated memory.
+ * \param [in] forest    A forest.
+ * NOTE: This will also free the memory of the scheme and the cmesh, since
+ *       the forest took ownership of them.
+ *       If we do not want this behaviour, but want to reuse for example the cmesh,
+ *       we need to call t8_cmesh_ref (cmesh) before passing it to t8_forest_new_uniform.
+ */
+static void
+t8_step2_destroy_forest (t8_forest_t forest)
+{
+  t8_forest_unref (&forest);
+}
+
+int
+main (int argc, char **argv)
+{
+  int                 mpiret;
+  sc_MPI_Comm         comm;
+  t8_cmesh_t          cmesh;
+  t8_forest_t         forest;
+  /* The prefix for our output files. */
+  const char         *prefix = "t8_step2_uniform_forest";
+  /* The uniform refinement level of the forest. */
+  const int           level = 3;
+  t8_locidx_t         local_num_elements;
+  t8_gloidx_t         global_num_elements;
+
+  /* Initialize MPI. This has to happen before we initialize sc or t8code. */
+  mpiret = sc_MPI_Init (&argc, &argv);
+  /* Error check the MPI return value. */
+  SC_CHECK_MPI (mpiret);
+
+  /* Initialize the sc library, has to happen before we initialize t8code. */
+  sc_init (sc_MPI_COMM_WORLD, 1, 1, NULL, SC_LP_ESSENTIAL);
+  /* Initialize t8code with log level SC_LP_PRODUCTION. See sc.h for more info on the leg levels. */
+  t8_init (SC_LP_PRODUCTION);
+
+  /* Print a message on the root process. */
+  t8_global_productionf (" [step2] \n");
+  t8_global_productionf
+    (" [step2] Hello, this is the step2 example of t8code.\n");
+  t8_global_productionf
+    (" [step2] In this example we build our first uniform forest and output it to vtu files.\n");
+  t8_global_productionf (" [step2] \n");
+
+  /* We will use MPI_COMM_WORLD as a communicator. */
+  comm = sc_MPI_COMM_WORLD;
+  /* Create the cmesh from step1 */
+  cmesh = t8_step2_build_prismcube_coarse_mesh (comm);
+
+  /* Build the uniform forest, it is automatically partitioned among the processes. */
+  forest = t8_step2_build_uniform_forest (comm, cmesh, level);
+  /* Get the local number of elements. */
+  local_num_elements = t8_forest_get_num_element (forest);
+  /* Get the global number of elements. */
+  global_num_elements = t8_forest_get_global_num_elements (forest);
+
+  /* Print information on the forest. */
+  t8_global_productionf (" [step2] Created uniform forest.\n");
+  t8_global_productionf (" [step2] Refinement level:\t\t\t%i\n", level);
+  t8_global_productionf (" [step2] Local number of elements:\t\t%i\n",
```

Comment:
```suggestion
  t8_productionf (" [step2] Local number of elements:\t\t%i\n",
```

Reply:
See the note in the Wiki on this: "Note that due to using t8_global_productionf opposed to t8_productionf only rank 0 will print its local number of elements."
I will leave it as is.